app/index.html

```diff
@@ -38,6 +38,22 @@
   <link integrity="" rel="stylesheet" href="{{rootURL}}assets/fe-redpencil.css">
 
   {{content-for "head-footer"}}
+
+  <!-- Matomo -->
+  <script type="text/javascript">
```

Comment:
Waarschijnlijk willen we het inserten van de hele blok ook conditioneel maken op environment. Je wil geen notificaties van alle devs die naar localhost aan het proxyen zijn ...

Reply:
Ik ben niet akkoord. Als we geavanceerde analytics willen testen moet het ook lokaal of op dev enabled zijn.
In matomo (of andere analytics providers) kan je op basis van de url de tracking anders afhandelen. 
Ik zou dit niet conditioneel doen. 
Ik zou zo dicht mogelijk tegen productie leunen voor alle omgevingen.